fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/mapping/RelationMapping.java

```diff
@@ -59,54 +61,84 @@ public static RelationMapping of(ImmutableBiMap<MappedRelation, MappedRelation>
      */
     public static List<RelationMapping> generate(List<CatalogRelation> sources, List<CatalogRelation> targets) {
         // Construct tmp map, key is the table qualifier, value is the corresponding catalog relations
-        LinkedListMultimap<Long, MappedRelation> sourceTableRelationIdMap = LinkedListMultimap.create();
+        HashMultimap<Long, MappedRelation> sourceTableRelationIdMap = HashMultimap.create();
         for (CatalogRelation relation : sources) {
             sourceTableRelationIdMap.put(getTableQualifier(relation.getTable()),
                     MappedRelation.of(relation.getRelationId(), relation));
         }
-        LinkedListMultimap<Long, MappedRelation> targetTableRelationIdMap = LinkedListMultimap.create();
+        HashMultimap<Long, MappedRelation> targetTableRelationIdMap = HashMultimap.create();
         for (CatalogRelation relation : targets) {
             targetTableRelationIdMap.put(getTableQualifier(relation.getTable()),
                     MappedRelation.of(relation.getRelationId(), relation));
         }
         Set<Long> sourceTableKeySet = sourceTableRelationIdMap.keySet();
-        List<List<Pair<MappedRelation, MappedRelation>>> mappedRelations = new ArrayList<>();
+        List<List<RelationMapping>> mappedRelations = new ArrayList<>();
```

Comment:
May be in the future, you can refactor code by using ImmutableEquivalenceSet here

Reply:
i think we should do it right now

Reply:
I try to use ImmutableEquivalenceSet, i found that `ImmutableEquivalenceSet` may be not suit the scene
such as I want to make a relation mapping as 
`RelationId#1 -> RelationId#2` which should keep the directivity。
after call` ImmutableEquivalenceSet.addEqualPair` then `tryToMap`,i found get the result
is 

> RelationId#1 -> RelationId#2

Reply:
I have discard `Pair<MappedRelation, MappedRelation>` and use `ImmutableBiMap` instead.